src/rpcblockchain.cpp

```diff
@@ -50,7 +50,7 @@ double GetDifficultyINTERNAL(const CBlockIndex* blockindex, bool networkDifficul
     int nShiftAmount = (powLimit >> 24) & 0xff;
 
     double dDiff =
-        (double)(powLimit & 0x00ffffff) / 
+        (double)(powLimit & 0x00ffffff) /
```

Comment:
Exclude this whitespace change from the PR. I'm not gonna block on the rest of them, as I'm guessing your editor is automatically stripping them, but this file is otherwise unaltered.

Reply:
Yeah, think it's my editor.